concepts/admin.py

```diff
@@ -11,47 +15,53 @@ def resolve(modeladmin, request, queryset):
         authorities.resolve(type(obj), obj)
 resolve.verbose_name = 'resolve selected concepts'
 
+class SeriesForm(forms.Form):
+        _selected_action = forms.CharField(widget=forms.MultipleHiddenInput)
+
 def merge_concepts(modeladmin, request, queryset):
-    """
-    This method gets called when we click on "Merge Concepts" action after we select the concepts
-    from the list of concepts
-    :param modeladmin:
-    :param request:
-    :param queryset:
-    :return:
-    """
-    
-    opts = modeladmin.model._meta
-    app_label = opts.app_label
-    RESOLVED_STATUS = Concept.RESOLVED
-    resolvedConceptCount = queryset.filter(concept_state = RESOLVED_STATUS).count()
-
-    #VGNWB-121 gets called only when there is only one resolved concept
-    if resolvedConceptCount == 1:
-        resolvedConceptsList = queryset.filter(concept_state = RESOLVED_STATUS).values()
-        unResolvedConceptsList = queryset.exclude(concept_state = RESOLVED_STATUS).values()
-
-        #As there will be only one element in the list
+
+        unResolvedConceptsList = queryset.exclude(concept_state = Concept.RESOLVED)
+        resolvedConceptsList = queryset.filter(concept_state = Concept.RESOLVED)
         resolvedConcept = resolvedConceptsList[0]
```

Comment:
When I select two concepts that are not resolved, and select the "merge concepts" action, I get an IndexError here.

![image](https://cloud.githubusercontent.com/assets/3451594/14181052/84c5bc96-f733-11e5-83b3-34d7e3ac169a.png)